tox.ini

```diff
@@ -925,3 +925,54 @@ changedir = {toxinidir}/packages/syftcli
 commands =
     pip list
     pytest
+
+[testenv:dev.k8s.registry]
+changedir = {toxinidir}/packages/grid
+passenv=HOME,USER
+allowlist_externals =
+    bash
+commands =
+    bash -c 'k3d registry create registry.localhost --port 12345 -v $HOME/.k3d-registry:/var/lib/registry'
+    bash -c '\
+        CYAN="\033[1;36m"; \
+        CLEAR="\033[0m"; \
+        printf "\n${CYAN}---------------------------------------------------------\n" ;\
+        printf "Make sure to add the following to your /etc/hosts file:${CLEAR}\n";\
```

Comment:
I think for windows users, it might be different from /etc/hosts

Reply:
We could add helper single line command like

for Linux/Mac
```sh
echo "127.0.01 k3d-registry.localhost" | sudo tee -a /etc/hosts
```

For windows:

Add-Content -Path "$env:systemroot\System32\drivers\etc\hosts" -Value "127.0.01 k3d-registry.localhost"

Reply:
Yes I was actually thinking of setting it up automatically, but since this command would be run frequently, you would end up seeing multiple entries like this:

<img width="1277" alt="image" src="https://github.com/OpenMined/PySyft/assets/11070218/e502f262-df7b-4590-a338-5d336376cc7d">

I'll create a script for this later but it would still need sudo (or run as admin on win) to actually patch the file. Moreover, WSL does not persist /etc/hosts/ patches because they are overwritten by host windows' hosts - so it will have a separate logic.